Documenting the CI Results Explorer (#11310)
DataDog/documentation: content/en/synthetics/troubleshooting/_index.md

{{< img src="synthetics/timeout.png" alt="API test on private location timing out" style="width:100%;" >}}

## CI/CD Testing

> ### I'm not seeing any CI metadata

Author: ```suggestion
### I'm not seeing any CI metadata in my CI Results Explorer
```

> It might be because you are using API endpoints to trigger your CI/CD test runs. To have your CI Results Explorer populated with CI metadata, you need to use the [NPM package][14].

Reviewer: ```suggestion
Check whether you are using API endpoints to trigger your CI/CD test runs. To have your CI Results Explorer populate with CI metadata, you must use the [NPM package][14].
```
